src/Collapse.js

```diff
@@ -14,7 +14,8 @@ const Collapse = React.createClass({
     fixedHeight: React.PropTypes.number,
     style: React.PropTypes.object, // eslint-disable-line react/forbid-prop-types
     springConfig: React.PropTypes.objectOf(React.PropTypes.number),
-    keepCollapsedContent: React.PropTypes.bool
+    keepCollapsedContent: React.PropTypes.bool,
+    onHeightReady: React.PropTypes.func
```

Comment:
For all non-required props it's good to have default value. If you add `onHeightReady: () => {}` then you don't need to do that extra check later in the code